name/templates/name/name_detail.html

```diff
@@ -1,210 +1,202 @@
-                <div class="row-fluid">
-                    <div class="span11">
-                        <h2 class="named">
-                            {{ requested_user.name }}
-                        </h2>
-                    </div>
-                    {% if user.is_authenticated %}
-                    <div class="btn-group span1">
-                        <a class="btn" href="{% url "admin:name_name_change" requested_user.id %}">
-                            <i class="icon-pencil"></i> Edit
-                        </a>
-                    </div>
-                    {% endif %}
-                </div>
-                <table class="table table-striped table-hover">
-                    <colgroup><col class="labels"><col class="data"></colgroup>
-                <tr>
-                    <th>Authorized:</th>
-                    <td >
-                        <!-- all schema "Things" have a name property -->
-                        <span itemprop="name">{{ requested_user.name }}</span>
-                    </td>
-                </tr>
-                <tr>
-                    <th>Name Type:</th>
-                    <td>{{ types|get_value:requested_user.name_type }}</td>
-                </tr>
-                <!-- current_LOCATION -->
-                {% for l in locations %}
-                <tr><th>Location: <em><small>({% if l.status == 0 %}current{% else %}former{% endif %})</small></em></th><td>{{ l.latitude }}, {{ l.longitude }}</td></tr>
-                {% endfor %}
-                <tr>
-                    <th>URI:</th>
-                    <td>
-                        <a href='{{ request.build_absolute_uri }}'>
-                            <span itemprop='url'>
-                                {{ request.build_absolute_uri }}
-                            </span>
-                        </a>
-                    </td>
-                </tr>
-                {% if requested_user.disambiguation %}
-                <tr>
-                   <th>Disambiguation:</th>
-                   <td>
+        {% endif %}
+
+        <h2 class="named">{{ requested_user.name }}</h2>
+
+        {% if user.is_authenticated %}
+            <a class="btn btn-default" href="{% url "admin:name_name_change" requested_user.id %}">Edit</a>
+        {% endif %}
+    </div>
+
+    <table class="table table-striped">
+        <colgroup><col class="labels"><col class="data"></colgroup>
+        <tr>
+            <th>Authorized:</th>
+            <td >
+                <!-- all schema "Things" have a name property -->
+                <span itemprop="name">{{ requested_user.name }}</span>
+            </td>
+        </tr>
+        <tr>
+            <th>Name Type:</th>
+            <td>{{ requested_user.get_name_type_label }}</td>
+        </tr>
+
+        {# Locations #}
+        {% for l in locations %}
+            <tr>
+                <th>Location: <em><small>({% if l.status == 0 %}current{% else %}former{% endif %})</small></em></th>

```

Comment:
Let's add _is_ methods to the Name model so that we don't have to do direct comparisons with an integer.